release-content/0.14/migration-guides/11426_Computed_State__Sub_States.md

```diff
@@ -1,7 +1,8 @@
-If the user accessed the NextState resource’s value directly or created them from scratch they will need to adjust to use the new enum variants:
+[`NextState`](https://docs.rs/bevy/latest/bevy/ecs/prelude/struct.NextState.html) is now an enum. If you were constructing it manually or matching on its value, you will need to use the equivalent enum variants.
 
-- if they created a `NextState(Some(S))` - they should now use `NextState::Pending(S)`
-- if they created a `NextState(None)` -they should now use `NextState::Unchanged`
-- if they matched on the `NextState` value, they would need to make the adjustments above
+|Before|After|
```

Comment:
In previous guides I've always tried to standardize to this instead of before/after

```suggestion
|0.13|0.14|
```